src/common/is-node-attached.js

```diff
@@ -1,11 +1,11 @@
 // Marionette.isNodeAttached
 // -------------------------
 
-import Backbone from 'backbone';
+import domMixin from '../mixins/dom';
 
 // Determine if `el` is a child of the document
 const isNodeAttached = function(el) {
-  return Backbone.$.contains(document.documentElement, el);
+  return domMixin.containsEl(document.documentElement, el);
```

Comment:
Thanks for looking into it.  I can't stand the `adown`s and `bup`s.

I'm cool with:
`return document.documentElement.contains(el && el.parentNode)`

@rafde ?